src/libecalc/presentation/yaml/yaml_types/components/legacy/energy_usage_model/yaml_energy_usage_model_consumer_system.py

```diff
@@ -65,6 +65,24 @@ class YamlCompressorSystemOperationalSetting(YamlBase):
         description="Set suction pressure equal for all consumers in a consumer system operational setting. \n\n$ECALC_DOCS_KEYWORDS_URL/SUCTION_PRESSURE",
     )
 
+    @model_validator(mode="after")
+    def ensure_increasing_cross_over(self):
+        if self.crossover is None:
+            return self
+        for index, crossover in enumerate(self.crossover):
+            if crossover >= max(self.crossover[: index + 1]) or crossover == 0:
```

Comment:
```suggestion
            if self.crossovers_exhausted(index, crossover) or self.has_no_crossovers():
```
or similar? more fluent and easier to read? too use more of those types of fluent typing/syntactic sugar?